src/test/java/com/byclosure/webcat/stepdefinitions/SeleniumInteractionsClickLink.java

```diff
@@ -0,0 +1,81 @@
+package com.byclosure.webcat.stepdefinitions;
+
+import com.byclosure.webcat.context.IContext;
+import com.byclosure.webcat.stepdefinitions.helpers.SeleniumInteractions;
+import org.junit.*;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.phantomjs.PhantomJSDriverService;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.io.File;
+import java.io.IOException;
+
+public class SeleniumInteractionsClickLink {
+
+    private static final String PAGES_URL_PATH = "http://localhost:63342/webcat-testing-platform/com/byclosure/webcat/stepdefinitions/";
```

Comment:
@vasco You're right, we did talk about using cucumber to test the steps. But since most of the steps' code is calling the API, I thought it would make more sense to unit test the API.
Also:

* we talked about having tests for the API
* I'm not getting how we can unit test the steps using cucumber